src/main/java/com/godaddy/icfp2017/Application.java

```diff
@@ -51,10 +62,13 @@ public static void main(String[] args) throws Exception {
       Socket skt = new Socket(host, port);
       gameDriver = new GameDriver(new BufferedInputStream(skt.getInputStream()),
                                   skt.getOutputStream(),
-                                  System.out,
+                                  debugStream,
                                   gameLogic);
     }
 
+    if (options.has("capture")) {
```

Comment:
nothing right now. I want to add a capture dump to make it easier to copy paste the log